config/initializers/elasticsearch.rb

```diff
@@ -1,7 +1,5 @@
 module Elasticsearch; end
 
-es_config = (YAML.load_file("#{Rails.root}/config/elasticsearch.yml") || {})[Rails.env]
-
-Tire::Configuration.url(es_config['url']) if es_config && es_config['url'].present?
+es_config = Rails.application.config.elasticsearch_config
```

Comment:
I prefer `Rails.application.config.elasticsearch`, the second `_config` is redundant.